go/vt/tlstest/tlstest_test.go

```diff
@@ -123,21 +124,20 @@ func TestClientServer(t *testing.T) {
 		t.Fatalf("TLSClientConfig failed: %v", err)
 	}
 
+	var serverErr error
 	wg.Add(1)
 	go func() {
 		// We expect the Accept to work, but the first read to fail.
 		defer wg.Done()
-		serverConn, err := listener.Accept()
-		if err != nil {
-			t.Fatalf("Connection failed: %v", err)
```

Comment:
The code that previously existed here seems confusing. Seems like it should have been failing the test if the read succeeds instead of writing to stdout?... Looks like the check for whether the Accept() failed is gone from the unit test with the current changes in the PR.

Reply:
Oh, I see now that serverErr is declared above, but it's not obvious at a glance because `:=` is used to assign it within a goroutine. Maybe it'd be clearer to pre-declare `var serverConn <type>` so `=` can be used instead of `:=` when assigning to `serverErr`?

Reply:
for review, yes, but should be fine for someone coming to the test without looking at this diff. Leaving it alone.